js/app/component_loader.js

```diff
@@ -18,6 +19,7 @@ export async function load_component({ api_url, name, id, variant }) {
 			component: c
 		};
 	} catch (e) {
+		console.error(e);
```

Comment:
Remove console error? Or maybe not

Reply:
We need to keep this one in so users can understand what might have one wrong for debugging purposes.